aws/table_aws_cloudtrail_trail_event_data_store.go

```diff
@@ -0,0 +1,184 @@
+package aws
```

Comment:
The filename looks incorrect, I think it should be `aws/table_aws_cloudtrail_event_data_store.go`